about/index.html

```diff
@@ -92,7 +91,7 @@ <h1 class="light section-title">Features</h1>
                     <div class="card-content-text">
                         <header class="feature-card-title">Open Source</header>
                         <p>Permissive Apache License allows using the framework in closed-source
-                            projects. You are welcome to contribute to the framework development.</p>
+                            projects. You are also welcome to contribute to our framework development!</p>
```

Comment:
Why do we want the exclamation in this sentence?

Reply:
Removed the exclamation mark at the end of the sentence.